### DISCUSSION TITLE
Expire Short Links Remove

### DISCUSSION THREAD

**Discussion by khalidzaibi** [ash-jc-allen/short-url#156] [Ideas]
if the deactivated_at date expired then automatically short links removed from database.

**Comment by ash-jc-allen** [comment]
Hey! I can see a use case for you wanting to do this, but I think this would be something that you'd need to implement in your own application.

Maybe through using the "pruning models" functionality that Laravel provides (https://laravel.com/docs/9.x/eloquent#pruning-models)? Or by writing your code to handle these being deleted from the database 🙂
